pkg/common/common.go

```diff
@@ -37,20 +36,31 @@ func HasRegexSpecialChars(input string) bool {
 	re := regexp.MustCompile(regexPattern)
 	return re.MatchString(input)
 }
-
-// c.FormValue("tags")
-func String2IntArray(str string) ([]int32, error) {
-	var array []int32
-	err := json.Unmarshal([]byte(str), &array)
-	if err != nil {
-		return nil, err
-	}
-	return array, nil
-}
-func GetFileName(file *multipart.FileHeader) string {
+func GetEncodeName(file *multipart.FileHeader) string {
 	id := uuid.New()
-	parts := strings.Split(file.Filename, ".")
-	fileType := parts[len(parts)-1]
-	newFileName := id.String() + "." + fileType
+	newFileName := id.String() + filepath.Ext(file.Filename)
 	return newFileName
 }
```

Comment:
somehow it didn't encode anything, maybe use `GenerateUniqueFileName` or `CreateUniqueName` would be more precise